README.md

```diff
@@ -349,6 +349,7 @@ Table of Contents
   * [sandstorm.io](https://sandstorm.io/) — Sandstorm is an open source operating system for personal and private clouds. Free plan offers 200 MB storage and 5 grains free
   * [gearhost.com](https://www.gearhost.com/pricing) — Platform for .NET and PHP apps. 256 MB of RAM for free on a shared server with limited resources
   * [glitch.com](https://glitch.com/) — Free unlimited public/private hosting for Node.js apps with features such as code sharing and real-time collaboration
+  * [flynn.io](https://flynn.io/) — A next generation open source platform as a service (PaaS). You can  setup and run your own Flynn cluster either locally, on dedicated hardware, or on a cloud provider
```

Comment:
unfortunately this one does not really fit the bill, we don't list things you have to self host/self manage - and if there is a managed offering (like flynn has) IT should have the free tier

Reply:
Ok. Feel free to decline my PR.